packages/server/lib/errors.coffee

```diff
@@ -352,7 +352,7 @@ API = {
         It looks like you are running the Cypress binary directly.
 
         This should only be done for DEVELOPMENT.
-        Please run Cypress via it's NPM package.
+        Please run Cypress via its NPM package.
```

Comment:
great catch, I hate reading something this, so sloppy

Reply:
I mean, I actually changed it to `it's` originally, so great catch on my own mistake? 😂

Reply:
the "catch" counts, the original authorship is murky 😅